README.md

```diff
@@ -107,7 +107,8 @@ These are the arguments that apply to all commands:
 2. Change the directory and build the binary
 
     ```bash
-    cd odf-cli/ && make build
+    cd odf-cli/
+    make
```

Comment:
```suggestion
    make build
```

Reply:
Why use `make build` when `make` wors?

I explained in the commit message why should use `make`:

```
- Simplify build instructions. `make build` works, but since it is the
  default target (for good reason) we don't need to specify it. This is
  also the expected usage.
```

Reply:
I agree it works but it is good practice to keep it complete so that new user don't get confused with only `make`

Reply:
But most users are used to build projects like this, this how all projects work for many decades:

```console
make 
make install
make test
```

Reply:
Sounds good to me, `make` is commonly understood as the short version of `make build`